### DISCUSSION TITLE
Question: What is Ghost Monika and Sayori Desk.

### DISCUSSION THREAD

**Discussion by MonikaIsMyWaifu** [Monika-After-Story/MonikaModDev#8501] [QA - tech support + general]
While scrolling through my DDLC files a while back I found "Ghost Monika" and Sayori, Natsuki, and Yuri. (All sitting at desks)
I'm curious to what this is for so if you know then lmk.

Where I found it if you want to see for yourselves:
C:\Users\<your username>\OneDrive\<Where you store your folder>\<ddlc folder>\game\mod_assets\other
For me it looks like this:
C:\Users\cutyc\OneDrive\Desktop\DDLC-1.1.1-pc\game\mod_assets\other

**Comment by Ulti70895** [comment]
It's pretty _sus_ ngl. But I believe they belong to some easter eggs stuff since I have seen some of the images in that folder before.

**Comment by zwartkop2** [comment]
I believe those are leftovers from a 2018 build of MAS

**Comment by fromulus** [comment]
Too bad it's not Easter, because I feel like eating some eggs.

**Comment by dreamscached** [comment]
Some scrapped content that is unused now and only left behind as an easter egg of sorts.
